src/databases/models/products.ts

```diff
@@ -1,111 +1,111 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable require-jsdoc */
-import { Model, DataTypes, Sequelize } from "sequelize";
-import sequelizeConnection from "../config/db.config";
-import { IProduct } from "../../types";
-
-class Products extends Model<IProduct> {
-    declare id: string;
-    declare shopId: string;
-    declare name: string;
-    declare description?: string;
-    declare price: number;
-    declare discount?: string;
-    declare category: string;
-    declare expiryDate?: Date;
-    declare expired: boolean;
-    declare bonus?: string;
-    declare images: string[];
-    declare quantity: number;
-    declare isAvailable: string;
-    declare createdAt: Date;
-    declare updatedAt: Date;
-
-    static associate(models: any) {
-        Products.belongsTo(models.Users, { foreignKey: "userId", as: "user" });

```

Comment:
We don't have this relationship  **`Products.belongsTo(models.Users, { foreignKey: "userId", as: "user" });`**